src/InvokePlaster.ps1

```diff
@@ -830,7 +830,7 @@ function Invoke-Plaster {
                 if (![string]::IsNullOrWhiteSpace($description)) {
                     $newModuleManifestParams['Description'] = $description
                 }
-                if (![string]::IsNullOrWhiteSpace($psVersion)) {
+                if (![string]::IsNullOrWhiteSpace($PowerShellVersion)) {
```

Comment:
Thanks for that info!  That will bring an end to C# developers laughing at me without telling why ;)